File: go/bubbleSort/bubbleSort_test.go
package bubbleSort

import (
	"testing"
	"reflect"
)

func TestCase1(t *testing.T) {
	test1:=[]int{5,3,1,2,4}
	expected:=[]int{1,2,3,4,5}
	output:=BubbleSort(test1)
	if !reflect.DeepEqual(output, expected){
		t.Errorf("BubbleSort output not correct")
	}
}

File: python/bubbleSort_test_.py
import unittest
from bubbleSort import bubbleSort

class TestProgram(unittest.TestCase):
  def test_case1(self):
    test1 = [5,2,1,3,4]
    output = bubbleSort(test1)
    self.assertEqual(output , [1,2,3,4,5])

if __name__ == "__main__":
  unittest.main()